packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -75,6 +80,10 @@ module.exports = {
         }
       }
 
+      if (orderBy) {
+        actionsOrder[orderBy as keyof ActionOrderByInput] =
+          orderDirection === Sort.desc ? orderDirection : Sort.asc
```

Comment:
Not totally following the purpose of this line of code. 

Why the `orderDirection === Sort.desc ? orderDirection : Sort.asc`? 

Instead should we just parse the provided `orderDirection` input into a `Sort` enum variant above and set that to the actionsOrder object?

Reply:
We can default to `Sort.desc` if no `orderDirection` was provided.

Reply:
cleaned it up a little bit, that was a no-brain kind of parsing the input :/